src/View/discipline/index.html.twig

```diff
@@ -0,0 +1,139 @@
+ {% extends "../layout.html.twig" %}
+    {% block stylesheet %}<link rel="stylesheet" href="/assets/css/discipline.css">{% endblock %}
```

Comment:
{% block stylesheet %}
        {{ parent() }}
        <link rel="stylesheet" href="/assets/css/discipline.css">
{% endblock %}
Using parent() to recieve also the style sheets from layout.html.twig, replace with the code above bro. Thanks

Reply:
{% block stylesheet %}{{ parent() }}<link rel="stylesheet" href="/assets/css/discipline.css">{% endblock %}
like that !